README.md

```diff
@@ -173,6 +173,32 @@ For [standalone installations](https://nix-community.github.io/home-manager/inde
   A: We primarily support the `unstable` branch, but try our best to support the current stable release.
   You can check if your stable release is currently supported at [status.nixos.org](https://status.nixos.org/)
 
+- Q: **"How do I fix the error "during evaluation because the option 'allow-import-from-derivation' is disabled"**\
+  A: Some ports require extra steps to configure the theme resulting in an [IFD](https://wiki.nixos.org/wiki/Import_From_Derivation).
```

Comment:
Currently https://wiki.nixos.org/wiki/Import_From_Derivation has no extra info about this error.

Reply:
we could link to https://nix.dev/manual/nix/latest/language/import-from-derivation.html instead

Reply:
Also does not provide extra info an the error, but has at least useful info instead of a blog and a dead link.

I am/was just hoping against better judgment that the official wiki will become better.